packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx

```diff
@@ -379,7 +414,7 @@ export class FileUploader {
         errors.push(this.maxFileSizeError);
       }
     }
-    this.errors = [...this.errors, ...errors];
+    this.errorText = errors.length ? errors[0] : '';
```

Comment:
Instead of just showing the first error, can we join the error message together with `,` separated and show it ?

Reply:
As per design, only one error message is shown here. Showing multiple errors in the same line, leads to scenarios we have not though of before. I think the current implmentation keeps it simple. Kindly reconsider.